admin/roles/class-abstract-role-manager.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * @package WPSEO\Admin\Roles
+ */
+
+/**
+ * Abstract Role Manager template.
+ */
+abstract class WPSEO_Abstract_Role_Manager implements WPSEO_Role_Manager {
+	/** @var array Registered roles. */
+	protected $roles = array();
+
+	/**
+	 * Registers a role.
+	 *
+	 * @param string      $role         Role to add.
```

Comment:
Make sure you're consist in the docs. `Role to add` should be changed to `Role to register`.

Reply:
we're*